robottelo/upgrade_utility.py

```diff
@@ -0,0 +1,165 @@
+"""Common Upgrade test utilities """
+
+from wait_for import wait_for
+from fabric.api import execute, run
+from robottelo.vm import VirtualMachine, settings
+from upgrade.helpers.docker import docker_execute_command
+from robottelo.api.utils import call_entity_method_with_timeout, entities
+
+
+class CommonUpgradeUtility(object):
+    """ CommonUpgradeUtility class for upgrade test cases """
+
+    def __init__(self, client_container_id=None):
+        self.client_container_id = client_container_id
```

Comment:
`client_container_id` might have made sense in first function that used code that is now refactored into `run_goferd`, but I believe it would be much better to call it `client_hostname`. After all, this is what this string represents - FQDN of host that we work with (execute commands, install software etc.).

Reply:
Right, fixed.